usaspending_api/recipient/delta_models/recipient_lookup.py

```diff
@@ -1,30 +1,686 @@
+            9 AS priority,
+            recipient_hash,
+            duns_recipient_hash,
+            awardee_or_recipient_legal AS legal_business_name,
+            NULL AS duns,
+            NULL AS uei,
+            source,
+            address_line_1,
+            address_line_2,
+            NULL AS business_types_codes,
+            city,
+            congressional_district,
+            country_code,
+            ultimate_parent_unique_ide AS parent_duns,
+            ultimate_parent_legal_enti AS parent_legal_business_name,
+            ultimate_parent_uei AS parent_uei,
+            state,
+            zip4,
+            zip5,
+            action_date AS update_date,
+            NULL AS alternate_names,
+            ROW_NUMBER() OVER (PARTITION BY recipient_hash ORDER BY action_date DESC, is_fpds, transaction_unique_id) AS row_num
+        FROM temporary_transaction_recipients_view
+        WHERE COALESCE(uei, awardee_or_recipient_uniqu) IS NULL
+    ),
+    union_all AS (
+        SELECT * FROM latest_duns_sam WHERE row_num = 1
+        UNION ALL
+        SELECT * FROM latest_tx WHERE row_num = 1
+        UNION ALL
+        SELECT * FROM latest_duns_sam_parent WHERE row_num = 1
+        UNION ALL
+        SELECT * FROM latest_tx_parent WHERE row_num = 1
+        UNION ALL
+        SELECT * FROM latest_duns_sam_no_name WHERE row_num = 1
+        UNION ALL
+        SELECT * FROM latest_tx_no_name WHERE row_num = 1
+        UNION ALL
+        SELECT * FROM latest_duns_sam_parent_no_name WHERE row_num = 1
+        UNION ALL
+        SELECT * FROM latest_tx_parent_no_name WHERE row_num = 1
+        UNION ALL
+        SELECT * FROM latest_tx_no_fabs_fpds WHERE row_num = 1
+    ),
+    union_all_priority AS (
+        SELECT
+            *,
+            ROW_NUMBER() OVER (PARTITION BY recipient_hash ORDER BY priority ASC, update_date DESC) AS row_num_union
+        FROM union_all
+    )
+    INSERT INTO temp.temporary_restock_recipient_lookup (
+        recipient_hash,
+        duns_recipient_hash,
+        legal_business_name,
+        duns,
+        uei,
+        address_line_1,
+        address_line_2,
+        business_types_codes,
+        city,
+        congressional_district,
+        country_code,
+        parent_duns,
+        parent_legal_business_name,
+        parent_uei,
+        state,
+        zip4,
+        zip5,
+        alternate_names,
+        source,
+        update_date,
+        row_num_union
+    )
+    SELECT
+        recipient_hash,
+        duns_recipient_hash,
+        legal_business_name,
+        duns,
+        uei,
+        address_line_1,
+        address_line_2,
+        business_types_codes,
+        city,
+        congressional_district,
+        country_code,
+        parent_duns,
+        parent_legal_business_name,
+        parent_uei,
+        state,
+        zip4,
+        zip5,
+        alternate_names,
+        source,
+        update_date,
+        row_num_union
+    FROM union_all_priority
```

Comment:
Should there be something that narrows down to top-priority items?
e.g. 
```
WHERE row_num_union = 1
```

Reply:
Or ... is this deferred, so you can generate alternate names from the full/redundant dataset? I see below you later delete where `row_num_union != 1`

Reply:
Mentioned this in the response above.